.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Run css_parser CI
+
+'on':
```

Comment:
```suggestion
on:
```

Reply:
Old habit tells me that YAML would convert it to `true`, heh. Done/pushed.